gradle.properties

```diff
@@ -8,10 +8,10 @@ antlrVersion=2.7.7
 awaitilityVersion=4.2.0
 findbugsVersion=3.0.2
 multithreadedtcVersion=1.01
-springBootVersion=3.2.0-RC1
+springBootVersion=3.2.0
 springShellVersion=1.2.0.RELEASE
-testcontainersVersion=1.17.6
-gemfireVersion=10.1.0
+testcontainersVersion=1.19.3
+gemfireVersion=10.1.0-build.1592
```

Comment:
wrong version

Reply:
Thanks. I'd put that there for testing and it slipped through.